### ISSUE TITLE
Compatibility with Nyx and Bloodmoon.

### ISSUE THREAD

**Issue by MagmaSlime123** [AnodeCathode/Time4TFC#2]
Title.

Bloodmoon adds, well, a Blood Moon event, which makes mobs spawn much more frequently, and closer to players.
Nyx also has a Blood Moon, but also has 2 other lunar events. A Harvest Moon and a Star Showers event.

Both mods' lunar events start on the normal Minecraft night cycle (You only need 1 of either mod, both cannot be used at once), though due to this mods shortened/extended day/night cycles, these lunar events don't sync with the Time4TFC day/night cycles.

Other mods are being used, but this can easily be replicated by just using Time4TFC, TFC, and Nyx or Bloodmoon.
Both Nyx and Bloodmoon have commands which allow you to force-start a Bloodmoon, (As said, both Bloodmoon and Nyx have Bloodmoons. Harvest Moons and Star Showers are a Nyx-only thing) Harvest Moon or Star Showers event.

Also reported this to Nyx's GitHub (Link to issue below), but they said it may have to be done on this mod's end. Not sure, though.

**Comment by AnodeCathode** [comment]
Yeah I can't imagine they're going to be compatible. No idea how I'd fix that. Would likely require knowing how Bloodmoon works. But I'm unlikely to figure it out myself. As we don't use Bloodmoon in TNFC and aren't likely to.

**Comment by MagmaSlime123** [comment]
Well, it'd be nice to have some sort of fix. Keep this open till you to try to figure this out.
